lib/types/src/schema/serverless.ts

```diff
@@ -3,11 +3,11 @@ import { z } from 'zod'
 export const ServerlessSchema = z.object({
   awsAccountId: z.string(),
   systemCode: z.string(),
-  region: z.string(),
+  region: z.array(z.string()).default(['eu-west-1']),
```

Comment:
**issue**: seeing as the option schema is changing this needs to be a breaking change denoted by a commit message with an exclamation mark after `feat(serverless)`. this isn't too bad seeing as we're doing prerelease versions atm so it (should) only jump to `0.2.0`.